joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/passes/ast/SimpleAstCreationPassTest.scala

```diff
@@ -112,4 +112,103 @@ class SimpleAstCreationPassTest extends RubyCode2CpgFixture {
       encoding.columnNumber shouldBe Some(5)
     }
   }
+
+  "Code field for simple fragments" should {
```

Comment:
As these tests are pretty much the same (in terms of assertions) as the ones in the previous group (line 9), I would suggest to add them there, instead of creating a new group ("Code field for simple fragments"). In other words, it's a needless distinction, right?